### ISSUE TITLE
Window too tall and narrow when opened from overflow menu in Firefox Proton

### ISSUE THREAD

**Issue by trinityd** [brandon1024/find#353]
## Issue Description
Window is too high and too narrow when opened from overflow menu in Firefox Proton (Dev edition, v90.0b12. Windows 10)

## Steps to Reproduce
- Install find in Firefox
- Put the extension in your overflow menu
- Open the extension, by clicking on it or with the shortcut

## Screenshots and Documentation
![image](https://user-images.githubusercontent.com/23157710/123501223-3273fd00-d5f8-11eb-8c0d-c1c9c37b6934.png)
![image](https://user-images.githubusercontent.com/23157710/123501231-3dc72880-d5f8-11eb-8db8-d79dc0760a61.png)

Looks fine if I take it out of the overflow menu:

![image](https://user-images.githubusercontent.com/23157710/123501554-a2838280-d5fa-11eb-89ba-b61d3a05bf31.png)

So not a huge deal. But as you can see I have a lot of great extensions like yours, so ideally I'd be able to leave it in the overflow menu and just use the shortcut.

This may be (probably is) a Firefox bug, just noticed it for the first time so figured I'd give you a heads up.

**Comment by redactedscribe** [comment]
I too do not want Find+ on my toolbar, but rather have it tucked away in the overflow menu. My toolbar already is quite crowded with the add-ons I need to click regularly. In my case, Find+ just takes up valuable space.

Ideally Ctrl+Shift+F would trigger the pop-up as shown in the last screenshot of the above post, no matter if the toolbar icon is visible or in the overflow menu.

**Comment by Chealer** [comment]
@the-nelsonator: Does this affect any current version?

**Comment by the-nelsonator** [comment]
@Chealer nope, seems like extensions can't be added to the overflow menu anymore, and it works as expected from the Extensions dropdown. Closing!
